.github/trigger_files/beam_PostCommit_Java_ValidatesRunner_Dataflow.json

```diff
@@ -1,4 +1,5 @@
 {
   "comment": "Modify this file in a trivial way to cause this test suite to run",
-  "modification": 1
+  "modification": 2,
```

Comment:
Technically this is only needed to force the testing infra to think the file is changed when you have no other changes to make.